resolv: NI_IDN unimplemented

 Some programs trying to use NI_IDN flag in case of it defined in netdb.h,
 so hide it to suppress bad values for ai_flags.
 Inspired by OpenWRT issue #12323

Signed-off-by: Leonid Lisovskiy <lly.dev@gmail.com>

diff --git a/include/netdb.h b/include/netdb.h
--- a/include/netdb.h
+++ b/include/netdb.h
@@ -663,7 +663,8 @@ struct addrinfo
 # define NI_NOFQDN	4	/* Only return nodename portion.  */
 # define NI_NAMEREQD	8	/* Don't return numeric addresses.  */
 # define NI_DGRAM	16	/* Look up UDP service rather than TCP.  */
-# ifdef __USE_GNU
+#if 0 /* uClibc: not implemented */
+/* # ifdef __USE_GNU */
 #  define NI_IDN	32	/* Convert name from IDN format.  */
 #  define NI_IDN_ALLOW_UNASSIGNED 64 /* Don't reject unassigned Unicode
 					code points.  */
